File: python/8kyu/grasshopper_messi_goals_function.py
"""

Author: MARCEL EBALLAR
GitHub: https://github.com/MACLEB1903

Source:
https://www.codewars.com/kata/55f73be6e12baaa5900000d4/train/python


Instructions:
Messi goals function

Messi is a soccer player with goals in three leagues: 
LaLiga
Copa del Rey
Champions

Complete the function to return his total number of goals in all three leagues.
Note: the input will always be valid.

For example:
5, 10, 2  -->  17

"""

def goals(*a):
	return sum(a)

File: typescript/8kyu/grasshopper-messi-goals-function.ts
/*

Author: MARCEL EBALLAR
GitHub: https://github.com/MACLEB1903

Source:
https://www.codewars.com/kata/55f73be6e12baaa5900000d4/train/typescript


Instructions:
Messi goals function
Messi is a soccer player with goals in three leagues:

LaLiga
Copa del Rey
Champions.
Complete the goals function to return his total goals (the sum) for all three leagues.

Note: the parameter for this function will always be a valid number.

Ex :

goals(5,10,2) == 5+10+2?: any = 17

*/

export function goals(): string[] {
  return [...arguments].reduce((a, b) => a + b, 0);
}
